test/common/plugin/currencyPlugin.fixtures/currencies/litecoin.ts

```diff
@@ -0,0 +1,272 @@
+      }
+    ],
+    'wif 2 - Non Compressed': [
+      '6vNUJU9VXjs79rr9jvqsoXFekNStYnYMnC287mnAovUhnTbA8cF',
+      {
+        metadata: {},
+        privateKeys: ['6vNUJU9VXjs79rr9jvqsoXFekNStYnYMnC287mnAovUhnTbA8cF']
+      }
+    ],
+    'wif 3 - Non Compressed': [
+      '6vusDCJqPM7KVAiMzXL8F27MWTfZ5sw8XJL1XT9wwgDMhfcugVQ',
+      {
+        metadata: {},
+        privateKeys: ['6vusDCJqPM7KVAiMzXL8F27MWTfZ5sw8XJL1XT9wwgDMhfcugVQ']
+      }
+    ],
+    'backwards compatible bip70': [
+      'litecoin:3EUxVRMAwPk2ZpBdiqRtY9uxWXSkgVCTFN?amount=0.11&r=https://merchant.com/pay.php?h%3D2a8628fc2fbe',
+      {
+        publicAddress: 'MLh6oJm8tWbTNKTXpiREMoAMqE3CbYzgi6',
+        // legacyAddress: '3EUxVRMAwPk2ZpBdiqRtY9uxWXSkgVCTFN',
+        metadata: {},
+        nativeAmount: '11000000',
+        currencyCode: 'LTC',
+        paymentProtocolUrl: 'https://merchant.com/pay.php?h=2a8628fc2fbe'
+      }
+    ],
+    'non backwards compatible bip70': [
+      'litecoin:?r=https://merchant.com/pay.php?h%3D2a8628fc2fbe',
+      {
+        metadata: {},
+        paymentProtocolUrl: 'https://merchant.com/pay.php?h=2a8628fc2fbe'
+      }
+    ]
+  },
+  encodeUri: {
+    'address only': [
+      { publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T' },
+      'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T'
+    ],
+    'legacy address1': [
+      { publicAddress: '34YtuLdPgopC8xNMKo98oJkGmeSQUrUWvc' },
+      '34YtuLdPgopC8xNMKo98oJkGmeSQUrUWvc'
+    ],
+    'legacy address2': [
+      { publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T' },
+      'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T'
+    ],
+    'legacy address3': [
+      { publicAddress: 'M816EdyuvWV7oETCfQeZb5mGWm9nHczijH' },
+      'M816EdyuvWV7oETCfQeZb5mGWm9nHczijH'
+    ],
+    'new format address': [
+      { publicAddress: 'MJiPwX84iBe4WnFDwsYGgtnz1XonPhUqhf' },
+      'MJiPwX84iBe4WnFDwsYGgtnz1XonPhUqhf'
+    ],
+    'address & amount': [
+      {
+        publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+        nativeAmount: '123456780000'
+      },
+      'litecoin:LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T?amount=1234.5678'
+    ],
+    // 'address, amount, and label': [
+    //   {
+    //     publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+    //     nativeAmount: '123456780000',
+    //     currencyCode: 'LTC',
+    //     metadata: {
+    //       name: 'Johnny Litecoin'
+    //     }
+    //   },
+    //   'litecoin:LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T?amount=1234.5678&label=Johnny%20Litecoin'
+    // ],
+    // 'address, amount, label, & message': [
+    //   {
+    //     publicAddress: 'LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T',
+    //     nativeAmount: '123456780000',
+    //     currencyCode: 'LTC',
+    //     metadata: {
+    //       name: 'Johnny Litecoin',
+    //       notes: 'Hello World, I miss you !'
+    //     }
+    //   },
+    //   'litecoin:LajyQBeZaBA1NkZDeY8YT5RYYVRkXMvb2T?amount=1234.5678&label=Johnny%20Litecoin&message=Hello%20World,%20I%20miss%20you%20!'
+    // ],
```

Comment:
Why is this uncommented out?

Reply:
I discovered why these parts of the fixture is commented out:

```
    ///
    // This is commented out because the parameter type for encodeUri
    // is `EdgeEncodeUri` and that type does not have a `metadata` property.
    // It looks like this was hacked into the old implementation, and hasn't
    // been implemented (hacked) into the new implementation.
    ///
```

Reply:
@thehobbit85 mentioned to just leverage a @ts-ignore here to keep the hack in place (instead of extending the type).